auth-backends/src/main/scala/bootstrap/rudder/plugin/AuthBackendsConf.scala

```diff
@@ -616,7 +616,7 @@ class RudderOidcUserService(
   // we need to use our copy of DefaultOAuth2UserService to log/manage errors
   super.setOauth2UserService(new RudderDefaultOAuth2UserService())
 
-  override val protocolName = "OIDC"
+  override val protocolName = "oidc"
```

Comment:
This is a human readable name used in debug/log, so this one should be as it is in the litterature. 
The problem is likely that we should not use that for the thing we put in the database.

Reply:
I think what we want is the key of the authentication provider stored in `RudderConfig.authenticationProviders` used for the authentication